plugins/basic/menu.py

```diff
@@ -0,0 +1,39 @@
+import inspect
+import os
+import sys
+
+
+def plugin_startup():
+    """
+    Initializes the Toolkit plugin for Nuke.
+    """
+
+    current_file_path = os.path.abspath(
```

Comment:
You caught me copy pasting code from Houdini, which is not required for Nuke/Nuke Studio. I've cleaned this up.